### DISCUSSION TITLE
Can I help contribute?

### DISCUSSION THREAD

**Discussion by Tedd-Codes** [minbrowser/min#2467] [General]
I would like to help contribute to min. I have some ideas and I know html/css/js and could help. But i dont know how to help. im new to GitHub and dint know how to help but if i can i will. i have posted some issues and helped contribute but i would also like to help with the development.

**Comment by Tedd-Codes** [comment]
The first thing i did was follow the development instructions and i added a error page report button.

**Comment by Tedd-Codes** [comment]
i figured it out but i still would like to be a collaborator

**Reply by Tedd-Codes** [reply]
@PalmerAL

**Reply by PalmerAL** [reply]
Do you have any ideas what area you'd like to work on? There isn't a big overarching plan, but the general goal is to add more functionality while keeping the UI minimal and not adding too much future maintenance burden. There's also always a lot of bugs open; most of them are hard to reproduce, but if you can find one that's reproducible those are always good to fix.
